modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/DriverInfo/index.tsx

```diff
@@ -63,6 +58,97 @@ const DriverInfo = () => {
 		}
 	};
 
+	const handleSaveChanges = (currentForm: any) => {
+		const hasAllRequiredFieldsToNextStep = currentForm.every(
+			(_form: any) =>
+				_form.firstName !== '' &&
+				_form.lastName !== '' &&
+				_form.relationToContact !== '' &&
+				_form.gender !== '' &&
+				_form.maritalStatus !== '' &&
+				_form.ageFirstLicenced !== '' &&
+				_form.occupation !== '' &&
+				_form.highestEducation !== '' &&
+				_form.millitaryAffiliation !== '' &&
+				_form.hasAccidentOrCitations !== ''
+		);
+
+		dispatch({
+			payload: true,
+			type: ACTIONS.SET_IS_ABLE_TO_SAVE,
+		});
+		dispatch({
+			payload: hasAllRequiredFieldsToNextStep,
+			type: ACTIONS.SET_IS_ABLE_TO_NEXT,
+		});
+	};
+
+	const handleSaveChangesWithOtherOccupation = (currentForm: any) => {
+		const hasAllRequiredFieldsToNextStep = currentForm.every(
+			(_form: any) =>
+				_form.firstName !== '' &&
+				_form.lastName !== '' &&
+				_form.relationToContact !== '' &&
+				_form.gender !== '' &&
+				_form.maritalStatus !== '' &&
+				_form.ageFirstLicenced !== '' &&
+				_form.occupation !== '' &&
+				_form.otherOccupation !== '' &&
+				_form.highestEducation !== '' &&
+				_form.millitaryAffiliation !== '' &&
+				_form.hasAccidentOrCitations !== ''
+		);
+
+		dispatch({
+			payload: true,
+			type: ACTIONS.SET_IS_ABLE_TO_SAVE,
+		});
+		dispatch({
+			payload: hasAllRequiredFieldsToNextStep,
+			type: ACTIONS.SET_IS_ABLE_TO_NEXT,
+		});
+	};
+
+	const handleSaveChangesWithGovernment = (currentForm: any) => {
+		const hasAllRequiredFieldsToNextStep = currentForm.every(
+			(_form: any) =>
+				_form.firstName !== '' &&
+				_form.lastName !== '' &&
+				_form.relationToContact !== '' &&
+				_form.gender !== '' &&
+				_form.maritalStatus !== '' &&
+				_form.ageFirstLicenced !== '' &&
+				_form.occupation !== '' &&
+				_form.governmentAffiliation !== '' &&
+				_form.highestEducation !== '' &&
+				_form.millitaryAffiliation !== '' &&
+				_form.hasAccidentOrCitations !== ''
+		);
+
+		dispatch({
+			payload: true,
+			type: ACTIONS.SET_IS_ABLE_TO_SAVE,
+		});
+		dispatch({
+			payload: hasAllRequiredFieldsToNextStep,
+			type: ACTIONS.SET_IS_ABLE_TO_NEXT,
+		});
+	};
```

Comment:
I believe that we can substitute the `.every` comparison for a single function that does that, maybe we could even turn these 3 functions into one. I would like to hear @silasbsi 's opinion on this.